Source/WebCore/style/StyleBuilderCustom.h

```diff
@@ -1890,7 +1890,7 @@ inline void BuilderCustom::applyValueFontSize(BuilderState& builderState, CSSVal
             break;
         }
     } else {
-        fontDescription.setIsAbsoluteSize(parentIsAbsoluteSize || !(primitiveValue.isPercentage() || primitiveValue.isFontRelativeLength()));
+        fontDescription.setIsAbsoluteSize(parentIsAbsoluteSize || !(primitiveValue.isPercentage() || primitiveValue.isFontRelativeLength()) || primitiveValue.isRems());
```

Comment:
Let’s consider making this check, `isPercentage || isFontRelativeLength || isRems`, a named function. That could give us an opportunity to write a comment about exactly why this is the correct check that covers all relative sizes.

That could either be a free function that we put in an appropriate header somewhere in the `style` directory, or a member function in the `CSSPrimitiveValue` class.

Reply:
@darinadler Do you have an idea of an appropriate name for a check covering `(!isPercentage && !isFontRelativeLength) || isRem`?

`isFontAbsoluteLength` isn't entirely accurate, because `rem` are part of `isFontRelativeLength`.

Reply:
Not sure I understand the situation perfectly, but maybe `isParentFontRelativeLength` is a good name. My general principle is to explain the concept to someone in plain language, then go back over the words I used and try to make a function name out of that.

Reply:
Another way to think of it is, we should write the comment explaining why the code needs to be this. Then convert that comment into a function name, and possibly omit the comment if the name makes everything clear.

Reply:
Thanks! That name fit perfectly and also made me find a related bug with the `rlh` unit.